angular-ngrx-scss/src/app/shared/components/tab-nav/tab-nav.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { selectProfileState } from 'src/app/state/profile/profile.selectors';
+
+@Component({
+  selector: 'app-tab-nav',
+  templateUrl: './tab-nav.component.html',
+  styleUrls: ['./tab-nav.component.scss'],
+})
+export class TabNavComponent {
+  profile$ = this.store.select(selectProfileState);
```

Comment:
It looks like there are only subtle differences between the tab nav between the user and org views of the profile page. Specifically "teams" and "people" get added for organizations, and "stars" gets removed for them too.

Reply:
yeah, seems to be the `NavComponent` is only used so far on the profile page, and reviewing the other apps is only used there. Probably @dustinsgoodman can clarify this but agreed if only we need to use it on the profile page, should not be in the `shared` folder, and should be below `profile` folder.

Reply:
I'm not Dustin but I think that would be the right course of action. I think it should be in the `profile` folder instead of `shared`, and if that's done then the code itself should be fine to remain as-is.

Reply:
I am Dustin and I think @Reshurum is right about the course of action I would take. 😄